agents/ibm_powervs/fence_ibm_powervs.py

```diff
@@ -18,15 +19,35 @@
 }
 
 def get_token(conn, options):
-        try:
-                command = "identity/token"
-                action = "grant_type=urn%3Aibm%3Aparams%3Aoauth%3Agrant-type%3Aapikey&apikey={}".format(options["--token"])
-                res = send_command(conn, command, "POST", action, printResult=False)
-        except Exception as e:
-                logging.debug("Failed: {}".format(e))
-                return "TOKEN_IS_MISSING_OR_WRONG"
-
-        return res["access_token"]
+	try:
+		if options["--token"][0] == '@':
+			key_file = options["--token"][1:]
+			try:
+				with open(key_file, "r") as f:
+				# read the API key from a file
```

Comment:
I would move this up a line, as the indentation suggests.

Reply:
done